Expecto/Expect.fs

```diff
@@ -262,7 +262,7 @@ let equal (actual : 'a) (expected : 'a) message =
       Tests.failtestf "%s. Actual value was %f but had expected it to be %f." message a e
   | a, e ->
     if actual <> expected then
-      if FSharpType.IsRecord(a.GetType(), BindingFlags.Default) then
+      if a <> null && FSharpType.IsRecord(a.GetType(), BindingFlags.Default) then
```

Comment:
Use `isNull` here, please.

Reply:
`Expect.isNull` hides `isNull`. 

Use: 
```fsharp
if Microsoft.FSharp.Core.Operators.isNull && FSharpType.IsRecord(a.GetType(), BindingFlags.Default) then
``` 
?

Reply:
Sure; you can make that a function alias above.